<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор фигур</string>
  </property>
  <widget class="GraphicsWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_figure_add"/>
   <addaction name="action_figure_remove"/>
   <addaction name="action_figure_move"/>
   <addaction name="action_figure_rotate"/>
   <addaction name="action_figure_22"/>
   <addaction name="action_figure_32"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Удалить все</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Удалить пересекающиеся</string>
   </property>
  </action>
  <action name="action_figure_22">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Фигура 22</string>
   </property>
   <property name="toolTip">
    <string>Фигура 22</string>
   </property>
  </action>
  <action name="action_figure_32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Фигура 32</string>
   </property>
   <property name="toolTip">
    <string>Фигура 32</string>
   </property>
  </action>
  <action name="action_figure_add">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить фигуру</string>
   </property>
  </action>
  <action name="action_figure_remove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Удалить фигуру</string>
   </property>
  </action>
  <action name="action_figure_move">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Переместить</string>
   </property>
  </action>
  <action name="action_figure_rotate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Повернуть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>graphicswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
